pkg/raft_cmdpb/raft_cmdpb.pb.go

```diff
@@ -71,7 +71,7 @@ func (x CmdType) String() string {
 	return proto.EnumName(CmdType_name, int32(x))
 }
 func (CmdType) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_raft_cmdpb_5dc25bc5f43956ae, []int{0}
+	return fileDescriptor_raft_cmdpb_8fa9c3cdf0171bc7, []int{0}
```

Comment:
Why this file is changed?

Reply:
I don't know. I cleaned and regenerate, but this file is also changed.

Reply:
I use protoc 3.8.0 following the README. Maybe the last commit didn't use the expected version?

Reply:
I think it's because https://github.com/pingcap/kvproto/pull/644 didn't generate the go proto.